packages/block-editor/src/components/rich-text/list-edit.native.js

```diff
@@ -6,7 +6,7 @@ import { Toolbar } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import {
 	changeListType,
-	getStartListFormat,
+	getLineListFormat,
```

Comment:
I meant to rename it as getLineNestingLevel, back when it was only about the nesting.  I think it's better to export two functions instead of overloading one for multiple purposes. https://github.com/WordPress/gutenberg/pull/14636/commits/66e971dd2bdfc34d86d6eb8b7e94cc2daa3a2939#r272474533

Reply:
Aha, I see. I will work on that.

Reply:
I pushed da0e1a7, splitting the function into separate ones and marking them as unstable. Let me know what you think @ellatrix , thanks!